src/locutus/model/user_input.py

```diff
@@ -59,8 +61,9 @@ def get_user_input(self, resource_type, collection_type, id, code, type):
         }
         """
         try:
+            mapped_pair=f'{code}|{mapped_code}'
```

Comment:
For situations like this, I would strongly recommend having a single function generate that paired string so that, if we ever have to change it, you only change the one place. And, that function could be useful in another module.